data/varad-kulkarni172.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Varad Kulkarni",
+  "bio": "Inquisitive Full Stack Web Developer who aims to make the paperless education more reliable and easy to use for people. I am also interested in Artificial Intelligence and Data Analytics."
```

Comment:
Please add a comma at the end of the line bio.

Reply:
> Please add a comma at the end of the line bio.

@Sindhuinti good spot! In future I would suggest adding this as an inline suggestion as this will then allow maintainers to commit the suggestion faster and merge the PR if there is nothing further that is required. If you are unsure as to how to do this you may wish to take a look at this [video](https://www.youtube.com/watch?v=QaB50I9vnVk).

Reply:
Thanks a lot !